package.json

```
@@ -20,8 +20,8 @@
     "test": "jest",
     "test:watch": "jest --collectCoverage=false --watch",
     "test:unit": "jest --collectCoverage=false",
-    "test:e2e": "cypress open",
-    "test:e2e:ci": "cypress run",
+    "test:e2e": "npx playwright test --headed",
+    "generate-e2e-tests": "npx playwright codegen localhost:8443/",
```

Fun fact: In `npm` scripts you don't need to use `npx` because `npm` scripts checks `node_modules/.bin` for executables before checking the current $PATH.

```suggestion
    "test:e2e": "playwright test --headed",
    "generate-e2e-tests": "playwright codegen localhost:8443/",
```

Also, what was the reason to run the tests `--headed` instead of headless? They run too quickly be default to be able to visually audit them... and we'd have to have a different script than the base one to run in CI (as it has to run headless there).

---

The reason for adding `--headed` was local testing, should have deleted it :)